migrations/5_BTCETHRebalancingSetToken.js

```diff
@@ -0,0 +1,212 @@
+require('module-alias/register');
+
+const SetProtocolUtils = require('set-protocol-utils');
+const BigNumber = SetProtocolUtils.BigNumber;
+const SetUtils = SetProtocolUtils.SetProtocolUtils;
+
+const BTCETHRebalancingManager = artifacts.require('BTCETHRebalancingManager');
+const Core = artifacts.require("Core");
+const LinearAuctionPriceCurve = artifacts.require('LinearAuctionPriceCurve');
+const RebalancingSetTokenFactory = artifacts.require("RebalancingSetTokenFactory");
+const SetTokenFactory = artifacts.require("SetTokenFactory");
+const WbtcMock = artifacts.require("StandardTokenMock");
+const WethMock = artifacts.require("WethMock");
+
+// Time constants
+const ONE_DAY_IN_SECONDS = new BigNumber(86400);
+const THIRTY_DAYS_IN_SECONDS = new BigNumber(2592000);
+
+// Unit and naturalUnit constants
+const DEFAULT_REBALANCING_NATURAL_UNIT = new BigNumber(10 ** 10);
+const ETH_DOMINANT_REBALANCING_NATURAL_UNIT = new BigNumber(10 ** 12);
+const DEFAULT_SET_NATURAL_UNIT = new BigNumber(10 ** 10);
+const DEFAULT_WBTC_UNIT = new BigNumber(1);
+const SET_FULL_TOKEN_UNITS = new BigNumber(10 ** 18);
+const WBTC_FULL_TOKEN_UNITS = new BigNumber(10 ** 8);
+const WETH_FULL_TOKEN_UNITS = new BigNumber(10 ** 18);
+
+// Decimal and price multiplier constants
+const DECIMAL_DIFFERENCE_MULTIPLIER = WETH_FULL_TOKEN_UNITS.div(WBTC_FULL_TOKEN_UNITS);
+const PRICE_PRECISION = new BigNumber(100);
+
+// Contract Addresses
+const WBTC_MEDIANIZER_ADDRESS_KOVAN = '';
+const WBTC_MEDIANIZER_ADDRESS_TESTRPC = '0x2002d3812f58e35f0ea1ffbf80a75a38c32173fa';
```

Comment:
how are you guaranteeing it deploys to this address? it should only be static like this if its on a snapshot